website/src/pages/index.astro

```diff
@@ -1,88 +1,134 @@
 
-    .description a {
-        @apply underline;
+    @media (max-width: 768px) {
+        .largerLogo img {
+            display: none;
+        }
+
+        .largerLogo div {
+            @apply mx-auto;
+        }
+        .largerLogo h1 {
+            @apply text-center text-6xl;
+        }
+
+        .largerLogo h2 {
+            @apply text-center text-2xl;
+        }
+
+        .infoSection h1 {
+            @apply text-2xl;
+        }
     }
 </style>
 
 <BaseLayout title='GenSpectrum Dashboards'>
-    <h1 class='mt-8'>Explore up-to-date pathogen genomic data</h1>
-    <div class='description mx-auto mb-8 max-w-[1400px] px-8'>
-        GenSpectrum…
+    <div class='largerLogo my-10 flex items-center justify-center'>
+        <img src='/virus-outline-colorful.svg' alt='colorful-logo' width='200' height='200' />
+        <div class='ml-10'>
+            <h1 class='mb-4 text-7xl'>
+                <BrandName fontsize='font-medium' />
+            </h1>
+            <h2 class='text-end text-3xl font-light text-slate-600'>Inspect pathogen genomic data</h2>
+        </div>
+    </div>
 
-        <ul class='ml-6 mt-2 list-disc'>
-            <li>
-                …is a fully interactive platform aiming to help scientists and public health professionals query and
-                analyze pathogen sequences.
-            </li>
-            <li>…is open source, the code is available on <a href='https://github.com/GenSpectrum'>GitHub</a>.</li>
-            <li>
-                …uses data from the <a href='https://www.insdc.org/'
+    <div class='infoSection mx-auto my-12 flex max-w-[1200px] px-8'>
+        <div class='ml-8 basis-1/2 text-balance'>
+            <div class='items-left my-8 flex'>
+                <div class='iconify size-7 mdi--information-outline'></div>
+                <h1 class='ml-2 text-3xl text-slate-600'>About Us</h1>
+            </div>
+            <span
+                >An interactive platform to help scientists and public health professionals query and analyze pathogen
+                sequences. The core team is at the <a href='https://bsse.ethz.ch/cevo'>cEVO group </a> at ETH Zurich, but
+                we are grateful for our many <a
+                    href='https://github.com/GenSpectrum/cov-spectrum-website?tab=readme-ov-file#contributors-'
+                >
+                    contributors
+                </a> from around the globe. We are funded by the <a href='https://ethz.ch/en.html'>ETH Zurich</a> and the
+                <a href='https://www.cdc.gov/'>CDC</a>.
+            </span>
+        </div>
+        <div class='ml-8 basis-1/2 text-balance'>
+            <div class='items-left my-8 flex'>
+                <div class='iconify size-7 mdi--wrench-outline'></div>
+                <h1 class='ml-2 text-3xl text-slate-600'>Open Source Tooling</h1>
+            </div>
+            <span
+                >Code is available on <a href='https://github.com/GenSpectrum'>GitHub</a>, with customizable components
```

Comment:
We changed the link in the footer, but apparently forgot the one in the text here - I think it would also be better to link the the repo instead of the project:
```suggestion
                >Code is available on <a href='https://github.com/GenSpectrum/dashboards'>GitHub</a>, with customizable components
```